brianlong/www.validators.app
# 182473504-p90-ping-thing-stat

#### What's this PR do?
- Add p90 stat to ping thing view

#### How should this be manually tested?
- use `rails r dev/create_test_ping_things.rb` to populate db if needed
- check the stats card at the top of the ping things page
- check web console for errors

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/182473504)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
